query/src/api/rpc/exchange/data_exchange.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_planners::Expression;
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub enum DataExchange {
+    // None,
+    Merge(MergeExchange),
+    ShuffleDataExchange(ShuffleDataExchange),
```

Comment:
Does that mean we haven't support broadcast exchange in current PR yet?

Reply:
Yes, this requires some work, and I will complete it in the next pr.